docs/contributing.md

```diff
@@ -227,33 +227,15 @@ result will look as expected when published to the website.
 
 ### Setting up the prerequisites
 
-To build the website, you must have Python and `poetry 1.8+` installed (the
-latest version is recommended). It is easiest to install `poetry` via `pipx`, as
-explained in the [Poetry installation instructions]. A few helpful points from
-the instructions: `pipx` can often be installed from your distribution, e.g.
-`sudo apt install pipx`; this will usually also install Python for you if
-necessary. Any version of `pipx` will do. If you are installing `pipx` manually,
-you may first need to follow the [Python installation instructions].
-
-[Python installation instructions]: https://docs.python.org/3/using/index.html
-[Poetry installation instructions]: https://python-poetry.org/docs/#installation
-
-Once you have `poetry` installed, you should ask it to install the rest
-of the required tools into a virtual environment as follows:
+To build the website, you must have `uv` installed. Please see: [Installing uv].
```

Comment:
I'd add "We currently support `uv` 0.5.1+. The CI also tests the latest version of `uv`".

Then, I'd add a comment "# If you update this, also update docs/contributing.md and docs.yml" inside `build.yml` next to the version.

Again, I can add this in a separate PR, or feel free to adjust my suggestion however seems reasonable.

Reply:
Separate PR is best b/c I don't have a test for the latest version of `uv` in this branch

Reply:
Update: I'll do it together w/ adding the latest-uv job